File: ReceiptRewards.PCL/Models/CommandIdentifier.cs

namespace ReceiptRewards.PCL.Models {

    public class CommandIdentifier {

        private int _commandIdentifierId;
        private int _commandElementId;
        private Code _code;
        private string _value;


        public int commandIdentifierId {
            get { return _commandIdentifierId; }
            set { _commandIdentifierId = value; }
        }

        public int commandElementId {
            get { return _commandElementId; }
            set { _commandElementId = value; }
        }

        public Code code {
            get { return _code; }
            set { _code = value; }
        }

        public string value {
            get { return _value; }
            set { _value = value; }
        }

    }
}

File: ReceiptRewards.WebServices/src/com/model/CommandIdentifier.java
package com.model;

import javax.xml.bind.annotation.XmlRootElement;

@XmlRootElement
public class CommandIdentifier {

    private int commandIdentifierId;
    private int commandElementId;
    private Code code;
    private String value;

    public CommandIdentifier() {
        this.commandIdentifierId = 0;
        this.code = null;
        this.value = "";
    }

    public CommandIdentifier(int commandIdentifierId) {
        this();
        this.commandIdentifierId = commandIdentifierId;
    }
    
    public CommandIdentifier(int commandIdentifierId, int commandElementId, Code code, String value) {
        this(commandIdentifierId);
        this.commandElementId = commandElementId;
        this.code = code;
        this.value = value;
    }

    public int getCommandIdentifierId() {
        return commandIdentifierId;
    }

    public void setCommandIdentifierId(int commandIdentiferId) {
        this.commandIdentifierId = commandIdentiferId;
    }

    public int getCommandElementId() {
        return commandElementId;
    }

    public void setCommandElementId(int commandElementId) {
        this.commandElementId = commandElementId;
    }

    public Code getCode() {
        return code;
    }

    public void setCode(Code code) {
        this.code = code;
    }

    public String getValue() {
        return value;
    }

    public void setValue(String value) {
        this.value = value;
    }
    
    @Override
    public String toString() {
    	// TODO: Implement
        return "";
    }
}
